### ISSUE TITLE
Better way to pass custom actions to headless app

### ISSUE THREAD

**Issue by DivadNojnarg** [Novartis/shinyValidator#11]
Currently, the way to pass custom script is a bit clunky:

```r
custom_horde <- "headless_app$set_inputs(obs = 200);
headless_app$set_inputs(obs = 1000);"
```

See whether we can't create a function with a set of expressions inside containing the script to play with the app...

**Comment by DivadNojnarg** [comment]
There is some progress but not yet fully satisfied. Notably, I'd like to get ride of the path param in the screenshot + `if (screenshot)` to avoid screenshot to occur during load test, profiling or reactlog:

```r
app$wait_for_idle()
      # Test new project
      app$click("new", wait_ = FALSE)
      if (screenshot) {
        app$get_screenshot(file.path(path, "2-new-project-modal.png"), delay = 3)
      }

      app$wait_for_idle()

      app$set_inputs(
        `project_manager-project_name` = "CI/CD project",
        `project_manager-project_description` = "This project was created by GitLab",
        `project_manager-project_visibility` = "private",
        #allow_no_input_binding_ = TRUE,
        wait_ = FALSE
      )
```

**Comment by DivadNojnarg** [comment]
Also fixed in #17
